cluster/ci/config/roles.yaml

```diff
@@ -656,3 +656,44 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: integr8ly
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: integr8ly-dev
+  users:
+  - mikenairn
+  - pmccarthy
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: admin
+    namespace: integr8ly
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: integr8ly-dev
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: admin
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: image-puller
+    namespace: integr8ly
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:unauthenticated
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
```

Comment:
is this duplicated with the above?

Reply:
The group above is unauthenticated, this is authenticated. The entire thing was duplicated from the last one to be added https://github.com/openshift/release/blob/master/cluster/ci/config/roles.yaml#L613-L658  

This looked to be doing the same thing that was required. Is there something needing changed?